starknet_devnet/server.py

```diff
@@ -101,11 +102,12 @@ def main():
 
     try:
         print(f" * Listening on http://{args.host}:{args.port}/ (Press CTRL+C to quit)")
+        pid = os.getpid()
```

Comment:
I would call this `main_pid`

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/337/commits/a350c050ab498a10c7d6decde837da54d002c320